components/layout/display_list_builder.rs

```diff
@@ -1717,6 +1717,12 @@ impl BlockFlowDisplayListBuilding for BlockFlow {
                                                       ScrollPolicy::Scrollable,
                                                       StackingContextCreationMode::Normal))
         } else {
+            match self.fragment.style.get_box().position {
+                position::T::static_ => {}
+                _ => {
+                    display_list.form_pseudo_stacking_context_for_positioned_content();
+                }
+            }
```

Comment:
nit: use `if ....position != position::T::static_ { ... }` maybe to save a level of indentation?